lib/components/rtsp-session/rtspsession-component.js

```diff
@@ -315,6 +324,7 @@ class RtspSessionComponent extends Component {
     this._sessionId && (message.headers.Session = this._sessionId)
     this._callHistory.push(options.method)
     if (!this._outgoingClosed) {
+      debug('msl:rtsp:outgoing')(`${message.method} ${message.uri} RTSP/1.0\n${objToStr(message.headers)}`)
```

Comment:
As mentioned above, should be better to do this debug log in the outgoing stream of the rtsp parser.